### PR TITLE
Generate Readme From Template

### PR DESCRIPTION
**Pull request by Vampire**
* Generate the README from a template on release and check for manual manipulation
* Some more README tweakage from our grammer communist
* made the quotes in the Gradle example consistent
* replaced the badges with links by markdown syntax and replaced the static version badge by "latest version on Maven Central" badge

### CODE REVIEW

File: .github/README_template.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+You can also login non-blocking asynchronously
+```java
+public class MyFirstBot {
+
+    public static void main(String[] args) {
+        // Read the token from the first program parameter when invoking the bot
+        String token = args[0];
+
+        new DiscordApiBuilder() .setToken(token) .login() .thenAccept(api -> {
```
Comment (Bastian) [inline .github/README_template.md]: I'm not sure why I missed it in the first PR, but these whitespaces shouldn't be there.

Comment (Kaleidox00) [reply .github/README_template.md]: make it a stylecheck thing

Comment (Bastian) [reply .github/README_template.md]: Stylechecks for the readme? ...

Comment (Kaleidox00) [reply .github/README_template.md]: stylecheck _all the things_

Comment (Bastian) [reply .github/README_template.md]: Please keep serious

Comment (Vampire) [reply .github/README_template.md]: Yeah, that's right, fixed.

File: .github/README_template.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+![](http://bastian-oppermann.de/javacord3_banner.png)
+# Javacord ![Latest version](https://shields.javacord.org/github/release/Javacord/Javacord.svg?colorB=brightgreen) [![Latest JavaDocs](https://shields.javacord.org/badge/JavaDoc-latest-yellow.svg?style=flat-square)](https://javadoc.javacord.org) [![Javacord Wiki](https://shields.javacord.org/badge/Wiki-Home-red.svg?style=flat-square)](https://github.com/Javacord/Javacord/wiki) [![Javacord Discord server](https://shields.javacord.org/discord/151037561152733184.svg?colorB=%237289DA&label=Discord&style=flat-square)](https://discord.gg/0qJ2jjyneLEgG7y3)
```
Comment (Bastian) [inline .github/README_template.md]: The badge for the latest version should have `?style=flat-square` to be consistent with the other badges.

Comment (Vampire) [reply .github/README_template.md]: Ups, my bad, fixed

**Pull request merged**